spec/controllers/lists_controller_spec.rb

```diff
@@ -35,6 +35,13 @@
       post :create, list: new_list
       expect(assigns(:list).errors.full_messages).to_not be_empty
     end
+
+    it "restricts non therapists from doing a create action" do
+      login_as_admin
+      post :create, list: {}
+      expect(response).to redirect_to(root_path)
+      expect(List.all.length).to eq(1)
```

Comment:
This spec is fragile because it depends on the `before :each` block staying the same - consider changing it to make a list with a specific name and check that no list with that name exists (or similar).